### ISSUE TITLE
Potential discussion

### ISSUE THREAD

**Issue by dajmcdon** [CDCgov/cfa-gam-rt#39]
## Provide links to references to methods or data sources

Hi @seabbs @zsusswein,

This is not really an issue, more an opening for discussion. I've been meaning to write since I had a Zoom with some of @kgostic's team earlier this summer. I suspect that Sam has seen https://github.com/dajmcdon/rtestim. There are a number of similarities with this project (also lots of differences), so I thought it might be helpful to discuss some of the synergies.

**Comment by zsusswein** [comment]
Hey @dajmcdon! 

Are you going to the [Rt collobathon](https://www.epistorm.org/collabathon2024)? Would love to chat more in-depth while we're there.

The short version: I think there's probably more synergy than overlap, though that may not currently be clear enough from the repo. One of the main goals here is to implement a hierarchical model at the local or facility level to make more robust real-time estimates at the state or national level. The focus on gams and splines here is that we need an approach that is lightweight in order to do this across the whole US in a big joint model. I'd love to figure out where you see synergy and what makes sense to think about more.

**Comment by seabbs** [comment]
Yes, definitely that would be great. As @zsusswein says in person at the Rt collabathon seems ideal if possible.
